community/kernel/src/main/java/org/neo4j/kernel/internal/DatabaseHealth.java

```diff
@@ -28,8 +28,8 @@
 
 public class DatabaseHealth
 {
-    private static final String panicMessage = "Database has encountered some problem, "
-            + "please perform necessary action (tx recovery/restart)";
+    private static final String panicMessage = "Critical error has encountered (see database logs for more details). " +
+            "Please restart the database.";
```

Comment:
It's at least missing a "been" in the sentence.

I would phrase it `The database has encountered a critical error, and needs to be restarted. Please see the debug.log file for details.`

Reply:
There is a small problem with referencing just 'debug.log' in a message since if you on a driver level you need a highlight that error is in server logs and not in a driver logs of any sort.
WDYT about: "The database has encountered a critical error, and needs to be restarted. Please see database logs for more details."?

Reply:
Sounds good 👍